planner/core/integration_test.go

```diff
@@ -989,6 +989,19 @@ func (s *testIntegrationSuite) TestApproxCountDistinctInPartitionTable(c *C) {
 	tk.MustQuery("select approx_count_distinct(a), b from t group by b order by b desc").Check(testkit.Rows("1 2", "3 1"))
 }
 
+func (s *testIntegrationSuite) TestApproxPercentile(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int)")
+	tk.MustExec("insert into t values(1), (2), (3), (4), (5)")
+	tk.MustQuery("select approx_percentile(a, 50) from t").Check(testkit.Rows("3"))
+	tk.MustQuery("explain select approx_percentile(a, 50) from t").Check(testkit.Rows("HashAgg_5 1.00 root  funcs:approx_percentile(test.t.a, 50)->Column#3",
```

Comment:
Please move the test data to testdata in json file.

Reply:
done